### PR TITLE
feat: make clientId mandatory

### PR DESCRIPTION
**Pull request by ieow**

### CODE REVIEW

**Comment by metalurgical** [comment]
Merge #69 first.

**Review by himanshuchawla009 — APPROVED** [review]
lgtm

File: Sources/CustomAuth/CustomAuth.swift [hunk 1]
```diff
@@ -66,15 +71,15 @@ open class CustomAuth {
...
+        self.init(web3AuthClientId: web3AuthClientId, aggregateVerifierType: aggregateVerifierType, aggregateVerifier: aggregateVerifier, subVerifierDetails: subVerifierDetails, network: network, loglevel: .debug, enableOneKey: enableOneKey, networkUrl: networkUrl)
     }
 
-    public convenience init(aggregateVerifierType: verifierTypes, aggregateVerifier: String, subVerifierDetails: [SubVerifierDetails], loglevel: OSLogType = .debug, enableOneKey: Bool = false, networkUrl: String? = nil) {
-//        let factory = CASDKFactory()
-        self.init(aggregateVerifierType: aggregateVerifierType, aggregateVerifier: aggregateVerifier, subVerifierDetails: subVerifierDetails, network: .legacy(.MAINNET), loglevel: loglevel, enableOneKey: enableOneKey, networkUrl: networkUrl)
-    }
+//    public convenience init(web3AuthClientId: String ,aggregateVerifierType: verifierTypes, aggregateVerifier: String, subVerifierDetails: [SubVerifierDetails], network: TorusNetwork, loglevel: OSLogType = .debug, enableOneKey: Bool = false, networkUrl: String? = nil) {
```
Comment (metalurgical) [inline Sources/CustomAuth/CustomAuth.swift:79]:
```suggestion
```

Comment (metalurgical) [inline Sources/CustomAuth/CustomAuth.swift:79]: We should remove these commented code blocks

**Review by metalurgical — APPROVED** [review]

**Pull request merged**